index.js

```diff
@@ -166,9 +178,10 @@ class ExecutorQueue extends Executor {
      * @param {Object} config.pipeline Pipeline of the job
      * @param {Object} config.job      Job object to create periodic builds for
      * @param {String} config.apiUri   Base URL of the Screwdriver API
+     * @param {String} config.eventId  Parent event ID (optional)
      * @return {Promise}
      */
-    async postBuildEvent({ pipeline, job, apiUri }) {
+    async postBuildEvent({ pipeline, job, apiUri, eventId }) {
```

Comment:
This might cause `Cannot destructure property` error if `eventId` is not there. Do we have a test case for this?

Reply:
as long as something other than `undefined` or `null` is passed into `postBuildEvent`, it should be fine.
In other words, if you are in favor of defensive coding, might as well give the argument default value of `{}`

Reply:
Looks like `config` will never be `null` from the jsdocs here, so maybe this is ok.